.github/actions/build-and-push-branch/action.yml

```diff
@@ -0,0 +1,56 @@
+name: "Build and Push Branch"
+description: "Build jars and docker images tagged for a particular branch. Primarily used for running OSS branch code in Cloud."
+inputs:
+  branch_version_tag:
```

Comment:
is there a way to auto generate this? if we're just doing `oss-branch-<commit hash>` could the action just generate it itself?

Reply:
The Cloud workflow generates this based on the input branch name, and then passes it in to the OSS action - I needed Cloud to be in charge of generating this because the same tag/version is used elsewhere in the cloud workflow, and the Cloud workflow is what initially receives the branch name input

Reply:
fair. you can make it optional fwiw. so if it is not set it just generates `oss-branch-<commit-hash>` but if it is set it uses what is passed in. probably not all that important though since it sounds like the only reason we would ever call this action is from the GH action in airbyte-cloud